rusoto/core/src/request.rs

```diff
@@ -153,8 +153,8 @@ impl Future for BufferedHttpResponseFuture {
     type Error = HttpDispatchError;
 
     fn poll(&mut self) -> Poll<Self::Item, Self::Error> {
-        self.future.poll().map_err(|err| err.into()).map(|async| {
-            async.map(|body| BufferedHttpResponse {
+        self.future.poll().map_err(|err| err.into()).map(|r#async| {
```

Comment:
Nit: Instead of using a raw identifier here, we should probably restructure this to use the `try_ready!` macro from `futures`; e.g:


```rust
let body = try_ready!(self.future.poll());

Ok(Async::Ready(BufferedHttpResponse {
    ...
}))
```

Reply:
Thanks for the pointer! Any changes aside from `edition = "2018"` in Cargo.toml was the work of `cargo fix` but it's good to know we can make it better.